src/js/content_scripts/input-mapper-utils.js

```diff
@@ -414,4 +406,94 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
             }
         }
     };
+
+    // TODO: Add tests for history navigation.
+
+    /**
+     *
+     * Navigate to the previous/next page in history using thumbsticks.
+     *
+     * @param {Object} that - The inputMapper component.
+     * @param {Integer} value - The value of the gamepad input.
+     * @param {Boolean} invert - Whether the history navigation should be in opposite
+     *                           order.
+     *
+     */
+    gamepad.inputMapperUtils.thumbstickHistoryNavigation = function (that, value, invert) {
+        // Get the updated input value according to the configuration.
+        var inversionFactor = invert ? -1 : 1;
+        value = value * inversionFactor;
+        if (value > 0) {
```

Comment:
You should really add a cutoff here instead of using zero.  I can get the thumbsticks on my Xbox One X controller to a resting state that's plus or minus `0.08` even.  My PS4 controller jitters constantly around its resting value, for example moving between 0.01 and 0.02 in whatever direction the thumbstick comes to rest in.

Reply:
We don't need to add the cutoff value here as even if these bypass the `0` value condition, they'll have to go through the `cutoffValue` when the `nextPageInHistory` ([here](https://github.com/fluid-lab/gamepad-navigator/pull/19/files/27188c6358e1b5c2dea7346eaee4120ec60b7bf5#diff-cc20e237566e63150ac1d82f0dbee3f8R476)) or `previousPageInHistory` ([here](https://github.com/fluid-lab/gamepad-navigator/pull/19/files/27188c6358e1b5c2dea7346eaee4120ec60b7bf5#diff-cc20e237566e63150ac1d82f0dbee3f8R443)) is called. So, in your case, if the value is plus/minus `0.08`, it would manage to call the `nextPageInHistory` or `previousPageInHistory` method, but the `cutoffValue` inside these methods will prevent anything from happening.

What I simply mean to say is we do have the `cutoffValue` as the check, but inside the method it calls. However, I can replace `0` with `cutoffValue` here as well if you want to keep it that way.

Reply:
I guess it's fine as long as the net effect is that jitter is screened out.  Down the road we may need to talk about doing this further upstream, i.e. protecting everything that is based on jittery controls at the point where we read it from the gamepad rather than requiring multiple cutoff checks downstream.  I'm happy to leave it for now, I'd rather you focus on the tab stability issues for now.